server/src/controllers/Opening.js

```diff
@@ -202,3 +203,62 @@ export const verifyPasscode = async (req, res) => {
     });
   }
 };
+
+export const getAllExistingOpenings = async (req, res) => {
+  try {
+    const openings = await Opening.find({});
```

Comment:
You can just get waiting openings, because we only take data of waiting Request and opening for insight, non-waiting is not insightful